app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, counter_cache: :comments_counter
```

Comment:
- [ ] Please add a method here that updates the number of comments as mentioned in the requirements. It can look something like this.
```
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```
def update_comments_counter
  post.increment!(:comments_counter)
end
```

Reply:
Done! 😄

Reply:
- Well resolved 👍🏽